src/styles/components/_dropdown.scss

```diff
@@ -1,6 +1,6 @@
 @use '../constants.scss';
 
-$dropdown-text: rgba(var(--spice-rgb-text), .7);
+$dropdown-text: var(--spice-text);
```

Comment:
It's using the RGB text for a reason here. The opacity is 0.7 because it lights up (to 1.0) on hover. You just removed that functionality.

Reply:
@theRealPadster Unfortunately a majority of themes don't support RGB text, as far as becoming less opaque on hover, I personally didn't really notice it that much and we can add another effect instead if you'd like. 

As far as the styling goes for the marketplace controls(search bar, scheme selector, settings icon, and theme dev tools button), I think that it'd be best to just use the same color variables so it doesn't look funky on some themes.

Reply:
Ah, fair. I didn't realize that people left out the RGB versions. Is it not standard?

Reply:
As far as it goes, the majority of themes uses color variables and the RGB is only injected where it was originally used on default Spotify, which is not a lot, mostly used for gradients.